Sources/SwiftDoc/Interface.swift

```diff
@@ -159,4 +159,24 @@ public final class Interface {
     public func defaultImplementations(of symbol: Symbol) -> [Symbol] {
         return relationshipsByObject[symbol.id]?.filter { $0.predicate == .defaultImplementationOf }.map { $0.subject }.sorted() ?? []
     }
+
+    // MARK: -
+
+    public func symbols(named name: String, resolvingTypealiases: Bool) -> [Symbol] {
+        var pathComponents: [String] = []
+        for component in name.split(separator: ".", omittingEmptySubsequences: true) {
```

Comment:
Nitpick: `omittingEmptySubsequences` defaults to true and can be omitted.

Reply:
```suggestion
        for component in name.split(separator: ".") {
```